kvirt/config.py

```diff
@@ -1338,15 +1339,18 @@ def plan(self, plan, ansible=False, url=None, path=None, autostart=False, contai
                 continue
             if k.disk_exists(pool, disk):
                 common.pprint("Disk %s skipped!" % disk, color='blue')
-                continue
+                disk_exists_on_client = True
             if len(vms) > 1:
                 shareable = True
             else:
                 shareable = False
-            newdisk = k.create_disk(disk, size=size, pool=pool, template=template, thin=False)
+            if disk_exists_on_client:
+                disk_to_add = k.disk_path(pool, disk)
```

Comment:
this would break on other platforms that libvirt since this method isnt defined elsewhere

Reply:
This have been addressed in the latest commit: We now contition on self.type == 'kvm' as suggested.